ksa_compliance/ksa_compliance/doctype/zatca_business_settings/zatca_business_settings.py

```diff
@@ -137,7 +136,8 @@ def onboard(self, otp: str) -> NoReturn:
         self._throw_if_api_config_missing()
 
         csr_result = self._generate_csr()
-        compliance_result, status_code = api.get_compliance_csid(self.fatoora_server_url, csr_result.csr, otp)
+        fatoora_server_url = self.fatoora_server_url.split(" | ")[1]
```

Comment:
Once we simplify the select field as indicated above, we can add a property called `fatoora_server_url` that returns the proper URL based on the select value instead of splitting strings everywhere.